webapp/src/components/expanded_view/component.tsx

```diff
@@ -242,6 +242,8 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                     skinTonePosition='search'
                     onEmojiSelect={this.handleUserPicksEmoji}
                     onClickOutside={this.toggleEmojiPicker}
+                    autoFocus={true}
+                    perLine={12}
```

Comment:
Does this make the picker shorter?

Reply:
it is wider

Reply:
I think we also wanted to make the picker shorter, so that it only shows about 4 lines at a time. It should be doable via CSS, like in this example: https://github.com/missive/emoji-mart/blob/main/packages/emoji-mart-website/example-custom-styles.html

Reply:
we could change the size of the emoji within the emoji picker to make it shorter if that's what we want.

@itao @tanmay-des any thoughts?

Reply:
We can also directly set the height to be shorter as well. But yeah, maybe we can wait until the design review to decide what is the best approach.